src/library/DIPS.Mobile.UI/Components/Pickers/DatePicker/HorizontalInLine/DateTimeView.cs

```diff
@@ -0,0 +1,117 @@
+using DIPS.Mobile.UI.Components.Content;
+using DIPS.Mobile.UI.Components.Content.DataTemplateSelectors;
+using DIPS.Mobile.UI.Converters.ValueConverters;
+using DIPS.Mobile.UI.Extensions.Markup;
+using Colors = DIPS.Mobile.UI.Resources.Colors.Colors;
+
+namespace DIPS.Mobile.UI.Components.Pickers.DatePicker.HorizontalInLine;
+
+public class DateTimeView : Grid
+{
+    public DateTimeView()
+    {
+        Loaded += CreateView;
```

Comment:
Koffer laga du layout i Loaded? Og du må unsubscribe :D

Reply:
Fordi den bugger seg om man gjør det i konstruktør pga horizontal content layout.

Reply:
Dette er bare kopiert fra `DateView`, sikkert for å fikse visuell bug på Android. Og den unsubscribes i den metoden ;) alt er uansett dratt ut i baseklasse nå